docs/reference/image_plugins.rst

```diff
@@ -0,0 +1,59 @@
+============================================
+Registering external image types via plugins
+============================================
+
+
+Usage
+~~~~~
+
+Toga can be extended, via plugins, to understand externally defined image types, gaining
+the ability to convert them to and from its own :any:`toga.Image` class.
```

Comment:
It might be worth calling out that PIL support in Toga is, technically, a plugin - it's just a plugin that is always installed.

Reply:
Done!